components/dialog/src/vwc-dialog.scss

```diff
@@ -71,3 +71,10 @@
 		width: 100%;
 	}
 }
+
+:host([topPosition]) {
+	.mdc-dialog {
+		top: var(--dialog-top-position);
```

Comment:
do you mean this?:
`$dialog-top-position: 100px; 
top: var(#{$dialog-top-position}, --dialog-top-position);`

Reply:
I mean be able to prevent arbitrary values. not sure what was the design decision eventually

![image](https://user-images.githubusercontent.com/15140652/153423346-9ac3c7a3-5663-42bb-a49c-ca9a3b964370.png)

Reply:
I've changed it to 4 predefined sizes